frontend/public/components/app.jsx

```diff
@@ -471,8 +480,13 @@ const PollConsoleUpdates = React.memo(function PollConsoleUpdates() {
     setPluginVersionsChanged(true);
   }
 
+  const consoleCapabilitiesChanged = !_.isEqual(
+    prevUpdateData?.capabilities,
+    updateData?.capabilities,
+  );
   const consoleCommitChanged = prevUpdateData?.consoleCommit !== updateData?.consoleCommit;
-  if (stateInitialized && consoleCommitChanged && !consoleChanged) {
+
+  if (stateInitialized && (consoleCommitChanged || consoleCapabilitiesChanged) && !consoleChanged) {
```

Comment:
I think the addition have be included in the story AC?

Reply:
yes, I've just noticed it when addressing your comment 👍